internal/driver/flamegraph.go

```diff
@@ -0,0 +1,363 @@
+// Copyright © 2017 Martin Spier <spiermar@gmail.com>
```

Comment:
Wrong license header?

Copyright owners need to be listed in the AUTHORS file, individual contributors are listed in CONTRIBUTORS.